src/types/common/contactPoint.ts

```diff
@@ -23,7 +22,7 @@ export interface ContactPoint {
   use: ContactPointUse;
 }
 
-export default () => ({
+export default (t: (key: string) => string) => ({
```

Comment:
```suggestion
export default contactPoint = (t: (key: string) => string)
```

Update import accordingly. The current version doesn't follow typescript best practices.